esbuild.package.js

```diff
@@ -0,0 +1,57 @@
+const fs = require('fs');
+const path = require('path');
+const esbuild = require('esbuild');
+const { clean } = require('esbuild-plugin-clean');
+const { copyFolderFiles, addReleaseFlag } = require('@hackolade/hck-esbuild-plugins-pack');
+const { EXCLUDED_EXTENSIONS, EXCLUDED_FILES, DEFAULT_RELEASE_FOLDER_PATH } = require('./buildConstants');
+
+const packageData = JSON.parse(fs.readFileSync('./package.json').toString());
+const RELEASE_FOLDER_PATH = path.join(DEFAULT_RELEASE_FOLDER_PATH, `${packageData.name}-${packageData.version}`);
+
+const copyTeradataClient = () => ({
+	name: 'copyTeradataClient',
+	setup(build) {
+		build.onEnd(async () => {
+			try {
+				await fs.promises.cp(
+					path.resolve(__dirname, 'reverse_engineering', 'addons', 'TeradataClient.jar'),
+					path.join(RELEASE_FOLDER_PATH, 'reverse_engineering', 'addons', 'TeradataClient.jar'),
```

Comment:
```suggestion
					path.join(RELEASE_FOLDER_PATH, 'addons', 'TeradataClient.jar'),
```
Tricky case but we have to use compatible path according to teradatClientPath: 
![image](https://github.com/hackolade/Teradata/assets/70570504/65752dbf-87f7-4d3c-9f16-bbb6e525d6e6)